app/tests/Feature/Http/VotingLists/JsonTest.php

```diff
@@ -0,0 +1,77 @@
+<?php
+
+use App\Enums\CountryEnum;
+use App\Enums\VoteResultEnum;
+use App\Group;
+use App\Vote;
+use App\Member;
+use App\VotingList;
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Support\Carbon;
+
+uses(Tests\TestCase::class, RefreshDatabase::class);
+
+beforeEach(function () {
+    $date = new Carbon('2021-05-23');
+    $display_title = 'Budged 2020';
+
+    $greens = Group::factory([
+        'code' => 'GREENS',
+        'name' => 'Greens/European Free Alliance',
+        'abbreviation' => 'Greens/EFA',
+    ])->create();
+
+    $this->greensFor = Member::factory([
+        'first_name' => 'Jane',
+        'last_name' => 'DOE',
+        'country' => CountryEnum::NL(),
+    ])->activeAt($date, $greens)->count(1);
+
+    $this->votingList = VotingList::factory([
+        'date' => $date,
+        'vote_id' => Vote::factory([
+            'result' => VoteResultEnum::REJECTED(),
+        ]),
+        'description' => 'Budged 2020',
+        ])
+        ->withMembers('FOR', $this->greensFor)
+        ->create();
+});
+
+it('creates a JSON with one row per member', function () {
+    $response = $this->get("/votes/{$this->votingList->id}.json");
+
+    $decoded = json_decode($response->content(), true);
+
+
+    $expected = [
+        'title' => 'Children’s rights',
+        'date' => '2021-05-23T00:00:00.000000Z',
+        'result' => 'rejected',
+        'members' => [
+            [
+                // member_id is autogenerated inside the database, because it is the first entry.
+                'member_id' => 1,
```

Comment:
Yes, in theory, this *should* be the case, as the test database *should* be reset before each test run. However, in the past, this led to problems. So I’d not use a constant ID here, but instead do something like this:

```suggestion
                // member_id is autogenerated inside the database, because it is the first entry.
                'member_id' => $this->votingList->members()->first()->id,
```